packages/dart/sshnoports/bundles/shell/install.sh

```diff
@@ -214,10 +217,52 @@ post_systemd_message() {
 
 install_systemd_unit() {
   unit_name="$1"
+  systemd_unit="$systemd_dir/$unit_name"
+  systemd_config="$systemd_unit.d/override.conf"
   no_mac
-  mkdir -p "$systemd_dir"
-  dest="$systemd_dir/$unit_name"
-  cp "$script_dir/systemd/$unit_name" "$dest"
+  if [ -f "$systemd_unit" ]; then
+    # migrate old config from systemd unit file to override.conf
```

Comment:
This has been the focus of my testing for the past few days - making sure that we can migrate existing config into the new approach.